js/components/root.js

```diff
@@ -95,15 +95,15 @@
 				}
 
 				if (this.alreadyOnLatest) {
-					return t('caniupdate', 'This app will show results again, once Nextcloud 14 is close.');
+					return t('caniupdate', 'This app will show results again, once Nextcloud 14 is closed.');
```

Comment:
`Close` as in `close to be done`.
Not as a word like `finished`

Reply:
I am thinking others, like me, will not catch that, in turn making for erroneous translations.
Would you accept "nearing release"?